### PR TITLE
Add screenshots for bookmark commands

### PR DESCRIPTION
**Pull request by siddarth2824**
Added screenshots with annotations for the following section and commands:
* Bookmark Expense section preface
* Bookmark Income section preface
* `AddBookmarkExpenseCommand`
* `AddBookmarkIncomeCommand`
* `EditBookmarkExpenseCommand`
* `EditBookmarkIncomeCommand`
* `ConvertBookmarkExpenseCommand`
* `ConvertBookmarkIncomeCommand`

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -528,9 +528,13 @@ Lists all incomes with the category `prize` and/or `gift`, with amounts between
 
 ### 4.5 Bookmark Expense
 
-Fine<span>$</span><span>$</span>e's Bookmark Expense feature is used to store frequent expenses that the user makes such as paying of monthly phone bills or buying bubble tea weekly.
+Fine\\$\\$e's Bookmark Expense feature is used to store frequent expenses that the user makes such as paying of monthly phone bills or buying bubble tea weekly.
 The user will be then be able to edit, delete and convert a bookmarked expense to conveniently add it into Fine<span>$</span><span>$</span>e's expense list.
 
+![Bookmark Expense Overview](images/userguide/bookmark/AnnotatedBookmarkExpenseOverview.png)
+
+Bookmark Expenses List Panel in the Expenses Tab
```
Comment (ianyong) [inline docs/UserGuide.md:536]: This line isn't very clear as to what it's for. :/

image

If it is meant to be captions to the image, perhaps we could work out some styling to use that makes its purpose more obvious?

Same for the other captions.

**Review by ianyong — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (ianyong) [reply docs/UserGuide.md:536]: To be resolved in a separate PR. Issue #256 opened.

**Review by ianyong — APPROVED** [review]
LGTM!

**Review by yongping827 — APPROVED** [review]
LGTM

**Pull request merged**